### PR TITLE
refactor: improve landing page loading performance

### PR DESCRIPTION
**Pull request by tonyhhyip**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] i18n translation has been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)
Improve landing page loading performance

Average Load: 1.50 s -> 1.10 s
* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -67,32 +66,34 @@ function dailyStats(t, props) {
...
 }
 
-const IndexPage = ({ data }) => {
+export default function IndexPage({ data }) {
   const { i18n, t } = useTranslation()
   const isSSR = typeof window === "undefined"
 
-  const latestStat = data.allDailyStats.edges[0].node
-  const remarksText = withLanguage(i18n, latestStat, "remarks")
+  const latestStat = React.useMemo(() => data.allDailyStats.edges[0].node, [
```
Comment (nandiheath) [inline src/pages/index.js:90]: what is the purpose of useMemo here?

**Review by nandiheath — CHANGES_REQUESTED** [review]
~i prefer KISS principle unless there is a proof of significant performance boost 🙂~
ah sorry I missed your message. so you mean the `useMemo` can reduce the number of rendering?

**Comment by tonyhhyip** [comment]
@nandiheath do you still need me to make changes?

**Review by nandiheath — APPROVED** [review]

**Comment by nandiheath** [comment]
> @nandiheath do you still need me to make changes?

sorry missed that 🤦‍♂️

**Pull request merged**